docs/node_benchmarks.md

```diff
@@ -0,0 +1,351 @@
+# Node Benchmarks for Scylla
+
+Whether you're running performance jobs or investigating an issue, you might
+want to now if the hardware Scylla is running meets you baseline perfromance
+expectations. That's what node benchmarks are for.
+
+### What benchmarks are we running?
+
+We're running:
+* sysbench - for CPU benchmarking. Github: https://github.com/akopytov/sysbench
+* cassandra-fio - for disk IO benchmarking. Github: https://github.com/KnifeyMoloko/cassandra-fio
+
+### How to run the benchmarks?
+
+Simply add the line `run_node_benchmarks: true` in your test case yaml.
+
+### At what point in the test are the benchmarks performed?
+
+Both benchmarks are performed at the end of adding nodes to the test cluster
+(nodes added after the cluster initialization are not included).
```

Comment:
Saying "the test cluster" you refer to some specific object. So, in current case it must be "a test Scylla cluster":

`Both benchmarks get performed at the end of "adding nodes" step to a test Scylla cluster, note that nodes which get added after the "cluster init" step won't be benchmarked.`

Reply:
Fixed (with a slightly different wording)